clang/docs/UndefinedBehaviorSanitizer.rst

```diff
@@ -214,13 +214,14 @@ Available checks are:
      the wrong dynamic type, or that its lifetime has not begun or has ended.
      Incompatible with ``-fno-rtti``. Link must be performed by ``clang++``, not
      ``clang``, to make sure C++-specific parts of the runtime library and C++
-     standard libraries are present.
+     standard libraries are present. The check is not a part of the ``undefined``
```

Comment:
Is it necessary to mention here that it's not part of the "undefined" group, instead of just in the check groups section below? There's several other checks in this list that aren't part of "undefined" but don't call out the group membership.

Reply:
Unnecessary, but probably usefull